setup.py

```diff
@@ -1,8 +1,16 @@
 from os.path import realpath, dirname, join
 from setuptools import setup, find_packages
-import biopandas
 
-VERSION = biopandas.__version__
+VERSION = None
+with io.open(
```

Comment:
shouldn't that be simply 

    with open

instead of 

    with io.open

??

Reply:
Yes, sorry. I failed a copy-paste.